src/components/reusable/tag/tag.ts

```diff
@@ -0,0 +1,129 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit-html/directives/class-map.js';
+import '@kyndryl-design-system/shidoka-foundation/components/icon';
+import clearIcon24 from '@carbon/icons/es/close/24';
+import clearIcon20 from '@carbon/icons/es/close/20';
```

Comment:
In Figma, it looks like the close icon is supposed to be the 16px size for both medium and small.

Reply:
Done.